spec/lib/account_spec.rb

```diff
@@ -0,0 +1,49 @@
+require './lib/bank_account/account'
+require 'spec_helper.rb'
+# root folder do rspec..
+describe BankAccount::Account do
+  let (:new_account200) {BankAccount::Account.new(100, 200)}
```

Comment:
I like the use of the let statement, keeps your tests nice and DRY!

You could maybe afford to give it an even more descriptive name, even if it's a little bit long.